examples/todomvc/src/models/todos.js

```diff
@@ -0,0 +1,48 @@
+import { types, destroy } from 'mobx-state-tree'
+
+const Todo = types.model(
+  {
+    text: 'Learn Redux',
+    completed: false,
```

Comment:
note: source code doesn't seem pretty printed. If you are using VSCode it should do so automatically, otherwise see the package json scripts on how to do that